addons/website_slides/static/src/js/public/components/course_tag_add_dialog/course_tag_add_dialog.js

```diff
@@ -0,0 +1,229 @@
+/** @odoo-module **/
+
+import { Component, useState } from "@odoo/owl";
+import { Dialog } from "@web/core/dialog/dialog";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { SelectMenu } from "@web/core/select_menu/select_menu";
+import { _t } from "@web/core/l10n/translation";
+import { uniqueId } from "@web/core/utils/functions";
+import { rpc } from "@web/core/network/rpc";
+
+export class CourseTagAddDialog extends Component {
+    static components = { Dialog, DropdownItem, SelectMenu };
+    static props = {
+        channelId: { type: Number, optional: true },
+        defaultTag: { type: String, optional: true },
+        tagIds: Array,
+        close: Function,
+    };
+    static template = "website_slides.CourseTagAddDialog";
+
+    async setup() {
+        super.setup();
+        this.title = _t("Add Tag");
+        this.choices = useState({
+            tagIds: [],
+            tagGroupIds: [],
+            tagId: null,
+            tagGroupId: null,
+        });
+        this.state = useState({
+            showTagGroup: false,
+            canCreateTagGroup: false,
+            canCreateTag: false,
+            alertMsg: "",
+        });
+        this.validation = useState({
+            tagIsValid: undefined,
+            tagGroupIsValid: undefined,
+        });
+        const [tags, groups] = await Promise.all([
+            this._fetch_choices("tag", [
```

Comment:
```suggestion
            this._fetchChoices("tag", [

Reply:
`About the close props, I just tested with the props removed and everything seems to work fine`:
Did you tried with the developer mode (with assets) mode on ?